File: lua/1010.lua
local a, a, b = io.read("*n", "*n", "*n")
local A, A, B = io.read("*n", "*n", "*n")
local amount = a*b + A*B;

print("VALOR A PAGAR: R$ " .. string.format("%0.2f", amount))

File: c/1010.c
#include <stdio.h>

int main()
{
    int a, b;
    double A, B;

    scanf("%d %d %lf", &a, &a, &A);
    scanf("%d %d %lf", &b, &b, &B);

    double amount = a*A + b*B;

    printf("VALOR A PAGAR: R$ %0.2lf\n", amount);

	return 0;
}

